cub/cub/device/dispatch/dispatch_histogram.cuh

```diff
@@ -859,6 +611,10 @@ public:
    * @param is_byte_sample
    *   Marker type indicating whether or not SampleT is a 8b type
    */
+  template <typename MaxPolicyT = typename ::cuda::std::_If<
+              ::cuda::std::is_void_v<PolicyHub>,
+              detail::histogram::policy_hub<SampleT, CounterT, NUM_CHANNELS, NUM_ACTIVE_CHANNELS, 0>,
+              PolicyHub>::MaxPolicy>
```

Comment:
Please retain the information given by the local variable names
```suggestion
  template <typename MaxPolicyT = typename ::cuda::std::_If<
              ::cuda::std::is_void_v<PolicyHub>,
              /* fallback policy hub */ detail::histogram::policy_hub<SampleT, CounterT, NUM_CHANNELS, NUM_ACTIVE_CHANNELS, /* isEven */ 0>,
              PolicyHub>::MaxPolicy>
```
Applies to 3 more places in this PR.